cpp_config/config.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
I think it will be better if we combine all the languages in the same config file that already exists in example, in that way it will be easier to run it later.